src/components/StorageOverview/TopConsumers/TopConsumers.js

```diff
@@ -90,30 +98,129 @@ const TopConsumersBody = ({ topConsumerStats }) => {
   return <div>{results}</div>;
 };
 
-export const TopConsumers = ({ topConsumerStats, topConsumerLoaded, LoadingComponent }) => (
-  <DashboardCard>
-    <DashboardCardHeader>
-      <DashboardCardTitle>Top Projects by Requested Capacity</DashboardCardTitle>
-    </DashboardCardHeader>
-    <DashboardCardBody isLoading={!topConsumerLoaded} LoadingComponent={LoadingComponent}>
-      <TopConsumersBody topConsumerStats={topConsumerStats} />
-    </DashboardCardBody>
-  </DashboardCard>
-);
+export class TopConsumers extends React.PureComponent {
```

Comment:
you need to pass specific props in https://github.com/kubevirt/web-ui-components/blob/c4331bf25e916fdb3dfd0dc671fc1f6eb19d8a78/src/components/StorageOverview/TopConsumers/TopConsumers.js#L228 not just `{...props}` otherwise `PureComponent` wont function as intended

Reply:
ack